pages/games/index.tsx

```diff
@@ -28,6 +28,7 @@ export enum STAGE {
   POSTGAME_LOBBY,
   GAME_OVER,
   COOP_GAME_OVER,
+  PLAY_GAME,
```

Comment:
This enum is for a different students game. I don't think you should be adding to it. I would create your own enum if needed.

Reply:
thank you! removed.